ibis/mapd/tests/test_client.py

```diff
@@ -38,6 +40,11 @@ def test_list_tables(con):
     assert len(con.list_tables(like='functional_alltypes')) == 1
 
 
+@pytest.mark.skipif(sys.version_info < (3, 6), reason='must have pymapd>=12')
```

Comment:
We need to `skipif` on the actual condition we care about which is that the `pymapd` version is `<0.12.0`. You can do the comparison using `pkg_resources.parse_version` like so:

```python
>>> from pkg_resources import parse_version
>>> import pymapd
>>> parse_version(pymapd.__version__) < parse_version('0.12.0')
```

Someone could install an older version of `pymapd` on Python 3.6, and we should skip this test then too.